specification/common/attribute-precedence.md

```diff
@@ -0,0 +1,176 @@
+# Attribute precedence on transformation to non-OTLP formats
+
+**Status**: [Experimental](../document-status.md)
+
+<details>
+<summary>Table of Contents</summary>
+
+<!-- toc -->
+
+- [Overview](#overview)
+- [Attribute hierarchy in OTLP messages](#attribute-hierarchy-in-otlp-messages)
+- [Precedence per Signal](#precedence-per-signal)
+  * [Traces](#traces)
+  * [Metrics](#metrics)
+  * [Logs](#logs)
+  * [Span Links, Span Events and Metric Exemplars](#span-links-span-events-and-metric-exemplars)
+- [Considerations](#considerations)
+- [Example](#example)
+- [Useful links](#useful-links)
+
+<!-- tocstop -->
+
+</details>
+
+## Overview
+
+This document provides supplementary guidelines for the attribute precedence
+that exporters should follow when translating from the hierarchical OTLP format
+to non-hierarchical formats.
+
+A mapping is required when flattening out attributes from the structured OTLP
+format, which has attributes at different levels (e.g., Resource attributes,
+InstrumentationScope attributes, attributes on Spans/Metrics/Logs) to a
+non-hierarchical representation (e.g., OpenMetrics labels).
+In the case of OpenMetrics, the set of labels is completely flat and must have
+unique labels only
+(<https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#labelset>).
+Since OpenTelemetry allows for different levels of attributes, it is feasible
```

Comment:
```suggestion
Since OpenTelemetry allows for different levels of attributes, it is possible
```
To me feasible sounds as if it was desirable and done on purpose. WDYT?